src/hooks/use_desmos_profile.ts

```diff
@@ -96,15 +102,19 @@ export const useDesmosProfile = (options: Options) => {
       });
     });
 
+    const connectionsWithoutNativeSorted = [...applications, ...chains].sort((a, b) => (
+      (a.network.toLowerCase() > b.network.toLowerCase()) ? 1 : -1
+    ));
+
     return ({
       dtag: profile.dtag,
       nickname: profile.nickname,
       imageUrl: profile.profilePic,
       coverUrl: profile.coverPic,
       bio: profile.bio,
-      connections: [...applications, ...chains].sort((a, b) => (
-        (a.network.toLowerCase() > b.network.toLowerCase()) ? 1 : -1
-      )),
+      connections: [nativeData, ...connectionsWithoutNativeSorted],
+      address: profile.address,
+      creationTime: profile.creationTime,
```

Comment:
We are not using this outside of the function:
```
const nativeData = {
      network: 'native',
      identifier: profile.address,
      creationTime: profile.creationTime,
    };
```

So i don't think an export is needed